doc/source/operations/cloudkitty.rst

```diff
@@ -0,0 +1,125 @@
+==========
+CloudKitty
+==========
+
+Configuring in kayobe-config
+============================
+
+By default, CloudKitty uses Gnocchi and Ceilometer as the collector and fetcher
+backends. Unless the system has a specific reason not to, we recommend instead
+using Prometheus as the backend for both. The following instructions explain
+how to do this.
+
+Enable CloudKitty and disable InfluxDB, as we are using OpenSearch as the
+storage backend. Set the following in ``kolla.yml``:
+
+.. code-block:: console
+
+  kolla_enable_cloudkitty: true
+  # Explicitly disable influxdb as we are using OpenSearch as the CloudKitty backend
+  kolla_enable_influxdb: false
+
+Set Prometheus as the backend for both the collector and fetcher, and Elasticsearch as the
+storage backend. Note that this is patched in our fork of CloudKitty to also
+work with OpenSearch. Proper support is still pending in Kolla-Ansible `here
+<https://review.opendev.org/c/openstack/kolla-ansible/+/898555>`__. If you have
+TLS enabled, you will also need to allow insecure connections for Prometheus
```

Comment:
I'd tried this before and was still seeing errors, afraid I didn't write them down. I'll see if I can recreate in a test env, as you say it seems like the defaults should work

Reply:
We have it working in at least one deployment

Reply:
Although that was explicitly setting the options in cloudkitty.conf rather than using those variables.

Reply:
I think I've found the issue, Kayobe did not have a value set for ``openstack_cacert`` when I tried to deploy because ``OS_CACERT`` was not set. I'm going to update the docs here to state that ``openstack_cacert`` should be set in Kayobe globals too.
On this point, I do wonder if we should alter defaults in SKC to handle this better.

Reply:
Slightly confusingly, `openstack_cacert` doesn't set `openstack_cacert` in kolla-ansible. Typically we set `openstack_cacert` in kolla-ansible as part of the internal TLS deployment.

Reply:
Confusing, but explains my problems. Thanks